File: applications/src/main/java/com/mydemo/applications/service/UserService.java
package com.mydemo.applications.service;

import com.mydemo.applications.entity.District;
import com.mydemo.applications.entity.JzUser;

import java.util.List;
import java.util.Map;

public interface UserService {
    List<District> findAddressPid(Integer id);
    JzUser findOne(int id);
    int addUser(JzUser user);
    int updateUser(JzUser user);
    JzUser login(String name,String pwd);
    Map findPersonal(Integer id);
}

File: applications/src/main/resources/templates/js/service/userService.js
app.service('userService',function($http){
    this.findAddressPid=function (pid) {
        return $http.get('../user/findAddressPid?pid='+pid);
    }
    this.findOne=function (id) {
        return $http.get('../user/findOne?id='+id);
    }
    this.updateUser=function (entity) {
        return $http.post('../user/updateUser?',entity);
    }
    this.addUser=function (entity) {
        return $http.post('../user/addUser?',entity);
    }
})